mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
```

Comment:
Here is the review:

The `req.params.orderId` should be validated and sanitized before being used in the `findByIdAndDelete` method to prevent potential security vulnerabilities such as SQL injection. Additionally, it would be better to use a more specific error message instead of the generic "No such order found" when the order is not found.